__test__/row.test.ts

```diff
@@ -172,6 +173,435 @@ describe("getBasecampTodosForHelpers", () => {
 
 });
 
+describe("getHelperGroups", () => {
+    it("should return an empty array when the row has not helpers value specified", () => {
+        const rowMock: Row = getRandomlyGeneratedRow();
+        const helpersValueMock: string = "";
+        rowMock.helpers = { value: helpersValueMock, hyperlink: null };
+
+        const { getHelperGroups } = require("../src/main/row");
+
+        const helperGroups: HelperGroup[] = getHelperGroups(rowMock);
+
+        expect(helperGroups).toStrictEqual([]);
+    });
+
+    it("should return a HelperGroup corresponding to each role when there are multiple roles", () => {
+        const rowMock: Row = getRandomlyGeneratedRow();
+        const helpersValueMock: string = "Food: John Doe, Jane Smith\nTech: Alice Johnson, Bob Brown";
+        rowMock.helpers = { value: helpersValueMock, hyperlink: null };
+
+        const memberMapMock: MemberMap = {};
+        memberMapMock["John Doe"] = getRandomlyGeneratedMember();
+        memberMapMock["John Doe"].gender = "Male";
+        memberMapMock["Jane Smith"] = getRandomlyGeneratedMember();
+        memberMapMock["Jane Smith"].gender = "Female";
+        memberMapMock["Alice Johnson"] = getRandomlyGeneratedMember();
+        memberMapMock["Alice Johnson"].gender = "Female";
+        memberMapMock["Bob Brown"] = getRandomlyGeneratedMember();
+        memberMapMock["Bob Brown"].gender = "Male";
+
+        const peopleToBasecampIdMap: { [name: string]: string } = {
+            "John Doe": randomstring.generate(),
+            "Jane Smith": randomstring.generate(),
+            "Alice Johnson": randomstring.generate(),
+            "Bob Brown": randomstring.generate(),
+        };
+
+        jest.mock("../src/main/members", () => ({
+            MEMBER_MAP: memberMapMock,
+            ALIASES_MAP: {},
+        }));
+
+        jest.mock("../src/main/groups", () => ({
+            GROUPS_MAP: {},
+        }));
+
+        jest.mock("../src/main/people", () => ({
+            getPersonId: jest.fn((personName) => peopleToBasecampIdMap.hasOwnProperty(personName) ? peopleToBasecampIdMap[personName] : randomstring.generate()),
+        }));
+
+        const expectedHelperGroups: HelperGroup[] = [
+            { role: "Food", helperIds: [peopleToBasecampIdMap["John Doe"], peopleToBasecampIdMap["Jane Smith"]] },
+            { role: "Tech", helperIds: [peopleToBasecampIdMap["Alice Johnson"], peopleToBasecampIdMap["Bob Brown"]] }
+        ];
+
+        const { getHelperGroups } = require("../src/main/row");
+
+        const helperGroups: HelperGroup[] = getHelperGroups(rowMock);
+
+        expect(helperGroups).toStrictEqual(expectedHelperGroups);
+    });
+
+    it("should apply filters to all members of a HelperGroup when a filter is included in the list of helpers", () => {
```

Comment:
Hmm, do we need to have filters that apply to the entire list of helpers for a role? I feel like filters should only be used in conjunction with an existing group or alias instead of to the entire list of helpers (as you already implemented). 

Personally, I don't think people would input sisters as part of the helpers and then go back to add a filter to remove them and that might cause confusion with people being unintentionally filtered out although in the cell it mentions sisters. 

Was there a scenario that you had in mind in which this would be used?

Reply:
@jkzang I think this was something Andrew mentioned unless I might have misunderstood. I guess it would be in the case where let's say sisters from UCSD, A2K, and IGSM were helping with something. Instead of specifying UCSD Sis, A2K Sis, IGSM Sis, you can just specify UCSD, A2K, IGSM, Sis. I see your point though

Reply:
@jkzang thanks for bringing this up, that's a good point. 
i think that for the helper column, since it's a hard rule to have everything be comma delimited, intuitively i think ppl would not assume a filter would apply past its comma limits. so let's not have filters apply to the whole list, as jack said.
however, it's true that the logic of having a filter apply to the whole list is what we'll be doing in the Who column. unfortunately this is inconsistent, it would've been nice to have the parsing logic be identical. in part, we don't rly have a choice since it's just one column and the multi-dropdown forces this, but at least it's still somewhat intuitive as ppl don't see commas, even though that's what the text ends up being when we parse it.

i may change my mind on this, but for now this is my thinking. lmk what you guys think.

Reply:
This makes sense especially to avoid confusion even if it's inconsistent. I'll remove the support for the filters in the comma separated filter list